src/jdk.sctp/unix/classes/sun/nio/ch/sctp/SctpMultiChannelImpl.java

```diff
@@ -745,17 +719,13 @@ private void removeAssociation(Association association) {
     }
 
     /**
-     * @throws  IllegalArgumentException
-     *          If the given association is not controlled by this channel
-     *
-     * @return  {@code true} if, and only if, the given association is one
-     *          of the current associations controlled by this channel
+     * @throws IllegalArgumentException If the given association is not controlled by this channel
```

Comment:
There should be a 1st sentence describing the function; its odd to see only an \@throws.